packages/mui/src/date-time-pickers/time-picker.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import { Meta, Story } from '@storybook/react';
+import { useState } from 'react';
+import { calendarDecorator } from 'src/@storybook/decorators';
+import { TimePicker, TimePickerProps } from './pickers';
+
+export default {
+  title: 'Time Picker/Time Picker',
+  decorators: [calendarDecorator()],
+  argTypes: {
+    color: {
```

Comment:
does it work? we should use theme-color-swapper

Reply:
I removed color argType from simple stories!